File: src/sdk-reference/java/1/document/count/snippets/count.java
try {
  int count = kuzzle.getDocument().count("nyc-open-data", "yellow-taxi", "{" +
    "\"query\": {" +
      "\"match\": {" +
        "\"license\": \"valid\"" +
      "}" +
    "}" +
  "}");

  System.out.println("Found " + count + " documents matching license:valid");
} catch (KuzzleException e) {
  System.err.println(e.getMessage());
}

File: src/sdk-reference/csharp/1/document/count/snippets/count.cs
try {
  int count = kuzzle.document.count("nyc-open-data", "yellow-taxi", @"{
    ""query"": {
      ""match"": {
        ""license"": ""valid""
      }
    }
  }");

  Console.WriteLine("Found " + count + " documents matching license:valid");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}
